amgut/handlers/human_survey_completed.py

```diff
@@ -19,6 +80,15 @@ def get(self):
 
             self.render('human_survey_completed.html', skid=self.current_user,
                         surveys=surveys)
+            consent_info = ag_data.getConsent(human_survey_id)
+            message = build_consent_form(consent_info)
+
+            try:
+                send_email(message, 'American Gut - Signed Consent Form(s)',
+                           recipient=consent_info['participant_email'])
+            except:
```

Comment:
I'm inclined to agree here. I _think_ `SMTPError` would just catch everything, but catching everything may be more useful as we can dump state that may be associated